src/templates/docs/text-transform/index.html

```diff
@@ -41,16 +41,16 @@ <h4 class="f4 mv0 fw6 dib mr4"><%= name %></h4>
       </article>
       <div class="ph3 ph5-ns pt4 pb5">
         <h2 class="f3 b">Examples</h2>
-        <h3 class="fw4 ttu mbn">Uppercase</h3>
-        <code class="code">.ttu </code>
-        <h3 class="fw4 ttl mbn">LOWERCASE</h3>
-        <code class="code">.ttl </code>
-        <h3 class="fw4 ttc mbn">capitalize multiple words</h3>
-        <code class="code">.ttc</code>
-        <h3 class="fw4 ttn mbn">TEXT Transform none</h3>
-        <code class="code">.ttn</code>
-        <h3 class="fw4 ttu tracked mbn">Uppercase with tracking</h3>
-        <code class="code">.ttu.tracked</code>
+        <code class="code gray">.ttu</code>
```

Comment:
Moved the class name usage above the example to keep consistent with rest of docs